src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -125,4 +126,35 @@ Integer getInt(Map<Option, ?> options) {
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
   // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
+
+  /**
+   * Creates a new lien which applies to the resource denoted by the parent field.
+   *
+   * @param lien
+   * @throws ResourceManagerException upon failure
+   */
+  Lien createLien(Lien lien);
+
+  /**
+   * Delete a Lien by `name`.
+   *
+   * @param name The name/identifier of the Lien to delete.
+   * @throws ResourceManagerException upon failure
+   */
+  void deleteLien(String name);
+
+  /**
+   * Get a Lien by `name`.
+   *
+   * @param name
+   * @throws ResourceManagerException upon failure
+   */
+  Lien getLien(String name);
```

Comment:
Get ->Gets 
@return

Reply:
same as above.

Reply:
all my comments are about javadoc
it should start with a verb which is not in the infinitive form: not 'Get' but 'Gets', not 'Return' but 'Returns'.

About @return tag, if you use @param and @throw you need @return.